devops/.env.sample

```diff
@@ -6,15 +6,10 @@ TERRAFORM_STATE_CONTAINER_NAME=tfstate
 IMAGE_TAG=__CHANGE_ME__
 ACR_NAME=__CHANGE_ME__
 
-# Service Principal used by the Composition Service to provision workspaces
-CONTRIBUTOR_SP_CLIENT_ID=__CHANGE_ME__
-CONTRIBUTOR_SP_CLIENT_SECRET=__CHANGE_ME__
-
 # Azure Resource Manager credentials used in CI/CD pipeline and local development scenarios
-ARM_TENANT_ID=__CHANGE_ME__
 ARM_SUBSCRIPTION_ID=__CHANGE_ME__
-ARM_CLIENT_ID=__CHANGE_ME__
-ARM_CLIENT_SECRET=__CHANGE_ME__
+RESOURCE_PROCESSOR_CLIENT_ID=__CHANGE_ME__
```

Comment:
I suppose there is a good reason for changing the name here, but isn't this SP used outside of the processor function too?

Reply:
Having variable ARM_CLIENT_SECRET confused terraform locally and it was deemed that since the SP is used in resource processor it should be marked as such. I have changed the variables accordingly in TF but not all the ways inside the chain. It works!